google/cloud/opentelemetry/internal/time_series.h

```diff
@@ -16,7 +16,7 @@
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_OPENTELEMETRY_INTERNAL_TIME_SERIES_H
 
 #include "google/cloud/version.h"
-#include <google/api/metric.pb.h>
+#include <google/monitoring/v3/metric_service.pb.h>
```

Comment:
Them rules suggest you need to keep both includes in this case.  And that may save you pain on imports too.

Reply:
Fine.